doc/source/serve/advanced-guides/advanced-autoscaling.md

```diff
@@ -24,10 +24,13 @@ It is always recommended to load test your workloads. For example, if the use ca
 As an example, suppose you have two replicas of a synchronous deployment that has 100ms latency, serving a traffic load of 30 QPS. Then requests are assigned to replicas faster than the replicas can finish processing them; more and more requests are queued up at the replica (these are considered "ongoing requests") as time goes on, and then the average number of ongoing requests at each replica steadily increases. Latency will also increase since new requests have to wait for old requests to finish processing. If you set `target_num_ongoing_requests_per_replica = 1`, Serve detects a higher than desired number of ongoing requests per replica, and adds more replicas. At 3 replicas, your system would be able to process 30 QPS with 1 ongoing request per replica on average.
 :::
 
-#### **max_concurrent_queries [default=100]**
-There is also a maximum queue limit that is proxies respect when assigning requests to replicas. The limit is defined by `max_concurrent_queries`. We recommend setting `max_concurrent_queries` to ~20 to 50% higher than `target_num_ongoing_requests_per_replica`. Note that `target_num_ongoing_requests_per_replica` should always be strictly less than `max_concurrent_queries`, otherwise the deployment never scales up. Take into account the following when setting `max_concurrent_queries`:
+#### **max_concurrent_queries [default=100] (DEPRECATED)**
+This has been renamed to `max_ongoing_requests`. `max_concurrent_queries` will be removed in a future release.
 
-- Setting it too low limits upscaling. For instance, if your target value is 50 and `max_concurrent_queries` is 51, then even if the traffic increases significantly, the requests will queue up at the proxy instead of at the replicas. As a result, the autoscaler only increases the number of replicas at most 2% at a time, which is very slow.
+#### **max_ongoing_requests [default=100]**
```

Comment:
are you still planning to update the default here?

Reply:
Yeah, I will update that in [another PR](https://github.com/ray-project/ray/pull/43004). This PR is purely for renaming.